src/test/java/CalculatorTest.java

```diff
@@ -19,4 +19,11 @@ public void testSubtract() {
     public void testMultiply() {
         assertEquals(6, calculator.multiply(2, 3), "2 * 3 sollte 6 ergeben");
     }
+
+    @Test
+    public void testDivide() {
+        assertEquals(2, calculator.divide(4, 2), "4 / 2 sollte 2 ergeben");
+        IllegalArgumentException exception = assertThrows(IllegalArgumentException.class, () -> calculator.divide(1, 0), "Teilen durch 0 sollte eine IllegalArgumentException werfen");
+        assertEquals("Cannot divide by zero.", exception.getMessage(), "IllegalArgumentException sollte Cannot divide by zero wiedergeben");
```

Comment:
Den divide by zero Fall könnte man besser in eine eigene Methode `testDivideByZero()` auslagern